src/reference/00-Getting-Started/04-Running.md

```diff
@@ -53,21 +53,23 @@ quotes. For example,
 In this example, `testOnly` has arguments, `TestA` and `TestB`. The commands
 will be run in sequence (`clean`, `compile`, then `testOnly`).
 
+**Note**: Running in batch mode requires JVM spinup and JIT each time, so your build may run slower. For day-to-day coding, we recommend using the sbt shell.
```

Comment:
We could add "or to use the **Continuous build and test** feature below, which works from the external command-line and also keeps the JVM running to allows fast re-builds"